protokube/pkg/protokube/helper.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package protokube
+
+import (
+	"fmt"
+	"os"
+)
+
+// touchFile does what is says on the tin, it touches a file
+func touchFile(p string) error {
+	_, err := os.Lstat(p)
+	if err == nil {
+		return nil
+	}
+
+	if !os.IsNotExist(err) {
+		return fmt.Errorf("error getting state of file %q: %v", p, err)
+	}
+
+	f, err := os.Create(p)
```

Comment:
Note to self: fix the race here :-)   `os.OpenFile(name, os.O_RDWR|os.O_CREATE|os.O_EXCL, 0644)`.